src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1601,8 +1601,18 @@ private MemberDeclarationSyntax ParseTypeDeclaration(SyntaxListBuilder<Attribute
 
                     return this.ParseClassOrStructOrInterfaceDeclaration(attributes, modifiers);
 
-                case SyntaxKind.RefKeyword:
                 case SyntaxKind.StructKeyword:
+                    // report use of "readonly struct" if feature is unsupported
+                    for (int i = 0, n = modifiers.Count; i < n; i++)
```

Comment:
will try.

---
In reply to: [118783089](https://github.com/dotnet/roslyn/pull/19713#discussion_r118783089) [](ancestors = 118783089)